### PR TITLE
Create Executive Vote - May 16, 2024.md

### PR DESCRIPTION
**Pull request by 0xldr**
TBD: 

- ~delegate compensation (pending JS handover)~
- ~Spark proxy spell (pending Phoenix Labs handover of the address)~
- ~Forum links for Keeper network update - JS to do~
- ~Add AD comp forum link - JS to do~

### CODE REVIEW

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: Template - [Executive Vote] Delegate Compensation, Whistleblower Bounty, Parameter Updates, Keeper Network Update, Launch Project Funding, Bug Bounty Payout, Spark Proxy Spell - May 16, 2024
+summary: Delegate Compensation for April 2024, whisleblower bounty and buffer payment for the offboarding of TrueName, stability fee and DSR updates, an update to a keeper network job, Dai and MKR funding for the Launch Project, payment of a bug bounty reward, execute Spark proxy spell.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:3]:
```suggestion
summary: Delegate compensation for April 2024, whistleblower bounty payment, buffer payment for the offboarding of TrueName, Stability Fee and DSR updates, an update to a keeper network job, Dai and MKR funding for the Launch Project, payment of a bug bounty reward, execute Spark proxy spell.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+The Governance Facilitators and Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:18]:
```suggestion
If this executive proposal passes, the following **changes** will occur within the Maker Protocol:

```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+- A total of TBD MKR will be distributed to TBD Aligned Delegates as compensation for May 2024.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:19]:
```suggestion
- A total of **TBD MKR** will be distributed to TBD Aligned Delegates as compensation for April 2024.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+- 20.84 MKR will be distributed as part of a whistleblower bounty.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:20]:
```suggestion
- 20.84 MKR will be distributed as a whistleblower bounty.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+- Multiple stability fees will be reduced, as will the Dai Savings Rate.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:21]:
```suggestion
- Multiple [Stability Fees (SF)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) will be reduced, as will the [Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate).
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+- A keeper network job will be updated.
+- 5,358,007 DAI and 1,969.17 MKR will be transferred to the Launch Project.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:23]:
```suggestion
- **5,358,007 DAI** and **1,969.17 MKR** will be transferred to the Launch Project.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+- A total of 55,000 DAI will be distributed as part of a Bug Bounty payment.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:24]:
```suggestion
- A total of **55,000 DAI** will be distributed as part of a Bug Bounty payment.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+- A Spark proxy spell will be triggered.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:25]:
```suggestion
- A Spark proxy spell will be executed.
```

Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:25]: (We should probably stick to one of the two, either "execute" or "trigger")

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:32]:
```suggestion
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+
+---
+
+## Proposal Details
+
+### Delegate Compensation for May 2024
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:38]:
```suggestion
### Delegate Compensation for April 2024
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+
+$executive_entry_description_1
+
+### Whistleblower Bounty
+
+* **Authorization:** [Atlas 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-operational-security)
+* **Proposal:** [Forum post](https://forum.makerdao.com/t/ad-derecognition-due-to-operational-security-breach-april-5-2024/24043)
+
+As a bounty for the information leading to the derecognition of TrueName delegate, the following transfer will be made, if this executive proposal passes:
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:47]:
```suggestion
As a bounty for the information leading to the derecognition of delegate TrueName, the following transfer will be made, if this executive proposal passes:
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+- `sequencer.addJob(0x2Ea4aDE144485895B923466B4521F5ebC03a0AeF)`
+
+The updated code may be seen in this GitHub [Pull Request](https://github.com/makerdao/dss-cron/pull/24).
+
+### Launch Project Funding
+
+* **Authorization:** [Accessibility Scope 9.1](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget)
+* **Proposal:** [Forum Post](https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/16)
+
+If this executive proposal passes the following tansfers will be made to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F):
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:95]:
```suggestion
If this executive proposal passes, the following tansfers will be made to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F):
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+
+- Transfer **5,358,007 DAI** to the Launch Project.
+- Transfer **1,969.17 MKR** to the Launch Project.
+
+### Bug Bounty Payout
+
+* **Authorization:** [Support Scope 13.1](https://forum.makerdao.com/t/bounty-payout-request-for-immunefi-bug-29806/24240/2)
+* **Proposal:** [Forum Post](https://forum.makerdao.com/t/bounty-payout-request-for-immunefi-bug-29806/24240)
+
+If this executive proposal passes the following transfers will be made as part of the Bug Bounty Program:
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:105]:
```suggestion
If this executive proposal passes, the following transfers will be made as part of the Bug Bounty Program:
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 14]
```diff
@@ -0,0 +1,131 @@
...
+- Transfer **5,000 DAI** to Immunefi at [0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18](https://etherscan.io/address/0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18)
+
+The post-mortem for this report may be found in this [forum post](https://forum.makerdao.com/t/post-mortem-for-immunefi-bug-report-29806/24239).
+
+### Spark Proxy Spell
+
+* **Authorization:** [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-13-under-sta-article-3-3/24250/2)
+* **Proposal:** [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-13-under-sta-article-3-3/24250)
+
+If this executive proposal passes a Spark Proxy Spell at [0x901E4450f01ae1A2615E384b9104888Cb9Cb02FF](https://etherscan.io/address/0x901E4450f01ae1A2615E384b9104888Cb9Cb02FF) will be triggered. It contains the following change:
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:117]:
```suggestion
If this executive proposal passes, a Spark Proxy Spell at [0x901E4450f01ae1A2615E384b9104888Cb9Cb02FF](https://etherscan.io/address/0x901E4450f01ae1A2615E384b9104888Cb9Cb02FF) will be triggered. It contains the following change:
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 15]
```diff
@@ -0,0 +1,131 @@
...
+### Parameter Updates
+
+#### [Stability Fee (SF)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) Changes
+
+* **Authorization:** [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-13-under-sta-article-3-3/24250/2)
+* **Proposal:** [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-13-under-sta-article-3-3/24250)
+
+If this executive proposal passes, as per the recommendation of the Stability Scope Advisory Council Member, BA Labs, the following SFs will be changed:
+
+- Decrease the ETH-A Stability Fee by 2 percentage points from 10.25% to **8.25%**.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:60]:
```suggestion
- Decrease the ETH-A SF by 2 percentage points from 10.25% to **8.25%**.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 16]
```diff
@@ -0,0 +1,131 @@
...
+- Decrease the ETH-B Stability Fee by 2 percentage points from 10.75% to **8.75%**.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:61]:
```suggestion
- Decrease the ETH-B SF by 2 percentage points from 10.75% to **8.75%**.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 17]
```diff
@@ -0,0 +1,131 @@
...
+- Decrease the ETH-C Stability Fee by 2 percentage points from 10.00% to **8.00%**.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:62]:
```suggestion
- Decrease the ETH-C SF by 2 percentage points from 10.00% to **8.00%**.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 18]
```diff
@@ -0,0 +1,131 @@
...
+- Decrease the WSTETH-A Stability Fee by 2 percentage points from 11.25% to **9.25%**.
```
Comment (blimpa) [inline governance/votes/Executive Vote - May 16, 2024.md:63]: And so on

**Review by blimpa — CHANGES_REQUESTED** [review]
Mostly minor stuff

File: governance/votes/Executive Vote - May 16, 2024.md (hunk 5 above)
Comment (0xldr) [reply governance/votes/Executive Vote - May 16, 2024.md:21]: links are provided lower down

File: governance/votes/Executive Vote - May 16, 2024.md (hunk 18 above)
Comment (0xldr) [reply governance/votes/Executive Vote - May 16, 2024.md:63]: not really necessary in our opinion so we won't be changing these

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 19]
```diff
@@ -0,0 +1,147 @@
...
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- A total of **208.09 MKR** will be distributed to ten Aligned Delegates as compensation for April 2024.
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - May 16, 2024.md:20]: The items should be ordered by judged impact/importance, imo Stability Scope Parameter changes should be first

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 20]
```diff
@@ -0,0 +1,147 @@
...
+### Whistleblower Bounty
+
+* **Authorization:** [Atlas 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-operational-security)
+* **Proposal:** [Forum post](https://forum.makerdao.com/t/ad-derecognition-due-to-operational-security-breach-april-5-2024/24043)
+
+As a bounty for the information leading to the derecognition of delegate TrueName, the following transfer will be made, if this executive proposal passes:
+
+- Transfer **20.84 MKR** to forum user Compacter at [0xbbd4bC3FE72691663c6ffE984Bcdb6C6E6b3a8Dd](https://etherscan.io/address/0xbbd4bC3FE72691663c6ffE984Bcdb6C6E6b3a8Dd).
+
+### Parameter Updates
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - May 16, 2024.md:67]:
```suggestion
### Stability Scope Parameter Updates
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 21]
```diff
@@ -0,0 +1,147 @@
+---
+title: Template - [Executive Vote] Delegate Compensation, Whistleblower Bounty, Parameter Updates, Keeper Network Update, Launch Project Funding, Bug Bounty Payout, Spark Proxy Spell - May 16, 2024
+summary: Delegate compensation for April 2024, whistleblower bounty payment, buffer payment for the offboarding of TrueName, Stability Fee and DSR updates, an update to a keeper network job, Dai and MKR funding for the Launch Project, payment of a bug bounty reward, execute Spark proxy spell.
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - May 16, 2024.md:3]:
```suggestion
summary: Aligned delegate compensation for April 2024, whistleblower bounty payment, buffer payment for the offboarding of TrueName, Stability Fee and DSR updates, an update to a keeper network job, Dai and MKR funding for the Launch Project, payment of a bug bounty reward, execute Spark proxy spell.
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 22]
```diff
@@ -0,0 +1,147 @@
+---
+title: Template - [Executive Vote] Delegate Compensation, Whistleblower Bounty, Parameter Updates, Keeper Network Update, Launch Project Funding, Bug Bounty Payout, Spark Proxy Spell - May 16, 2024
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - May 16, 2024.md:2]:
```suggestion
title: Template - [Executive Vote] Aligned Delegate Compensation, Whistleblower Bounty, Parameter Updates, Keeper Network Update, Launch Project Funding, Bug Bounty Payout, Spark Proxy Spell - May 16, 2024
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 23]
```diff
@@ -0,0 +1,147 @@
...
+summary: Delegate compensation for April 2024, whistleblower bounty payment, buffer payment for the offboarding of TrueName, Stability Fee and DSR updates, an update to a keeper network job, Dai and MKR funding for the Launch Project, payment of a bug bounty reward, execute Spark proxy spell.
+date: 2024-05-16T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Delegate Compensation, Whistleblower Bounty, Parameter Updates, Keeper Network Update, Launch Project Funding, Bug Bounty Payout, Spark Proxy Spell - May 16, 2024
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - May 16, 2024.md:8]:
```suggestion
# [Executive Proposal] Aligned Delegate Compensation, Whistleblower Bounty, Parameter Updates, Keeper Network Update, Launch Project Funding, Bug Bounty Payout, Spark Proxy Spell - May 16, 2024
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 24]
```diff
@@ -0,0 +1,147 @@
...
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Delegate Compensation for April 2024
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - May 16, 2024.md:38]:
```suggestion
### Aligned Delegate Compensation for April 2024
```

File: governance/votes/Executive Vote - May 16, 2024.md [hunk 25]
```diff
@@ -0,0 +1,147 @@
...
+| JAG         | 7.17         | [0x58D1ec57E4294E4fe650D1CB12b96AE34349556f](https://etherscan.io/address/0x58D1ec57E4294E4fe650D1CB12b96AE34349556f) |
+| Byteron     | 6.72         | [0xc2982e72D060cab2387Dba96b846acb8c96EfF66](https://etherscan.io/address/0xc2982e72D060cab2387Dba96b846acb8c96EfF66) |
+| UPMaker     | 3.58         | [0xbB819DF169670DC71A16F58F55956FE642cc6BcD](https://etherscan.io/address/0xbB819DF169670DC71A16F58F55956FE642cc6BcD) |
+
+### Whistleblower Bounty
+
+* **Authorization:** [Atlas 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-operational-security)
+* **Proposal:** [Forum post](https://forum.makerdao.com/t/ad-derecognition-due-to-operational-security-breach-april-5-2024/24043)
+
+As a bounty for the information leading to the derecognition of delegate TrueName, the following transfer will be made, if this executive proposal passes:
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - May 16, 2024.md:63]:
```suggestion
As a bounty for the information leading to the derecognition of AD TrueName, the following transfer will be made, if this executive proposal passes:
```

**Review by VoteWizard — APPROVED** [review]
Good job @MakerJanSky

**Pull request merged**